raspi/systemd/ss-susi-register.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Register the smart speaker online
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+Type=oneshot
+WorkingDirectory=/home/pi/SUSI.AI
+ExecStart=/usr/bin/python3 susi_installer/raspi/access_point/register.py
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Is this part necessary? Wouldn't it trigger an automatic activation of the unit when someone logs in?

Reply:
No, as far as I know, `WantedBy=multi-user.target` doesn't activate unit when someone logs in. So it should be fine.